spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusProperties.java

```diff
@@ -116,6 +118,21 @@ public Pushgateway getPushgateway() {
 		 */
 		private Map<String, String> groupingKey = new HashMap<>();
 
+		/**
+		 * Enable publishing via a Prometheus Pushgateway with Basic Auth.
+		 */
+		private Boolean authEnabled = false;
+
+		/**
+		 * Basic Auth Username.
+		 */
+		private String authUsername;
```

Comment:
We usually use a plain `username` for this.

Reply:
@snicoll  all lower case?

Reply:
Yes. As I mentioned, this is what we usually do so you should find a number of similar cases searching the codebase.

Reply:
@snicoll I'm aware of that, but I found other variables like `baseUrl`\`pushRate`\`groupingKey`, key word is uppper case?

Reply:
> you should find a number of similar cases searching the codebase.

All properties that represents a username in this codebase are defined as `username`. So, yes, as described in my first comment.

Reply:
@snicoll  so you advise `authusername` and `authPassword` ?

Reply:
a little wired

Reply:
No. As mentioned in my first comment, `username` (and `password`).

Reply:
very well, done